rmd2redmine.sh

```diff
@@ -0,0 +1,180 @@
+#! /usr/bin/bash
+
+
+htmlFile=""
+
+while getopts ':l:f:p:' opt; do
+  case $opt in
+    f)
+      htmlFile="$OPTARG"
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      exit 1
+      ;;
+    :)
+      echo "Error: -${OPTARG} requires an argument."
+      exit 1
+      ;;
+  esac
+done
+
+echo "fileName = commandArgs(trailingOnly = T)[1]
+
+if (!require('stringr')) {
+  install.packages('stringr')
+}
+if (!require('xml2')) {
+  install.packages('xml2')
+}
+if (!require('rvest')) {
+  install.packages('rvest')
+}
+if (!require('base64enc')) {
+  install.packages('base64enc')
+}
+
+library(stringr)
+library(xml2)
+library(rvest)
+library(base64enc)
+
+reportName = str_remove(fileName, '\\\\.html')
+
+# Set up helper functions
+df_to_textile = function(x) {
+  result = ''
```

Comment:
the default in R is to use '<-' instead of '=' so e.g. Bioconductor requires this

Reply:
(and people in general expect it :D)